### PR TITLE
Bugfix/native bandwidth cap

### PR DESCRIPTION
**Pull request by improbable-danny**
Removing bandwidth cap for gyms to be a fair comparison with currently how Spatial works

### CODE REVIEW

File: Game/Config/DefaultEngine.ini [hunk 1]
```diff
@@ -89,4 +89,12 @@ PhysXTreeRebuildRate=10
 DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,bUseMBPOuterBounds=False,MBPBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPOuterBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPNumSubdivs=2)
 ChaosSettings=(DefaultThreadingModel=DedicatedThread,DedicatedThreadTickMode=VariableCappedWithTarget,DedicatedThreadBufferMode=Double)
 
+; For NFR tests, we should adjust these once we have Spatial limits. This is to make a fair comparison between a playable game (without lag + banding)
+[/Script/OnlineSubsystemUtils.IpNetDriver]
+bNoTimeouts=true
```
Comment (m-samiec) [inline Game/Config/DefaultEngine.ini]: This is a lot of values - you sure we need them all?

Comment (improbable-danny) [reply Game/Config/DefaultEngine.ini]: I believe these are server settings and the ones below are for clients

Comment (m-samiec) [reply Game/Config/DefaultEngine.ini]: To rate-limit RPCs?

**Comment by ImprobableVlad** [comment]
Have you tested these changes? What numbers do they give? I'm curious about the impact, but it would also validate that these changes indeed take effect.

Also, on a kind of related note, do game developers generally stay below the default limit? I'm wondering whether the necessity for this is due to its contrived nature. If real games actually don't show this kind of activity, would it instead not be appropriate to adjust simplayer behaviour?

File: Game/Config/DefaultEngine.ini (hunk 1 above)
Comment (improbable-danny) [reply Game/Config/DefaultEngine.ini]: Yeah, those numbers are distributed to players when they join a server

**Comment by improbable-danny** [comment]
> Have you tested these changes? What numbers do they give? I'm curious about the impact, but it would also validate that these changes indeed take effect.
> 
> Also, on a kind of related note, do game developers generally stay below the default limit? I'm wondering whether the necessity for this is due to its contrived nature. If real games actually don't show this kind of activity, would it instead not be appropriate to adjust simplayer behaviour?

@ImprobableVlad 

The egress rises slightly (about 20%). Without these changes there is teleporting and loads of lag (some simplayers were completely starved and had no activity). Games probably wouldn't uncap this like we have but we know we have plenty of bandwidth available on the server - really we want to know how much bandwidth is used to have a stable playable experience with our gym (comparable to Spatial). In the future we will bring these numbers back down and implement a similar feature for Spatial. 

I suspect any medium/large scale game will hit the defaults so most probably will need to tweak it at some point, there are few options to deal with it such as the replication graph or using the dynamic GameNetworkManager which will adjust the available bandwidth to clients. Perhaps we should try this at some point also as well

File: Game/Config/DefaultEngine.ini [hunk 2]
```diff
@@ -89,4 +89,12 @@ PhysXTreeRebuildRate=10
 DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,bUseMBPOuterBounds=False,MBPBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPOuterBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPNumSubdivs=2)
 ChaosSettings=(DefaultThreadingModel=DedicatedThread,DedicatedThreadTickMode=VariableCappedWithTarget,DedicatedThreadBufferMode=Double)
 
+; For NFR tests, we should adjust these once we have Spatial limits. This is to make a fair comparison between a playable game (without lag + banding)
```
Comment (ImprobableVlad) [inline Game/Config/DefaultEngine.ini]:
```suggestion
; For NFR tests. This is to make a fair comparison between GDK and Native Unreal for a playable game (without lag + banding)
; TODO: We should adjust these once we have Spatial limits.
```

**Pull request merged**